partner_label/models/res_config_settings.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017-2019 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    partner_labels_width = fields.Float(
```

Comment:
Was there a reason not to put the settings directly in res.config.settings, but have them in company and then use related?

Reply:
`res.config.settings` is a transient model, we can't use this for permanent storage of our parameters. And making it related to the current company gives you storage very conveniently without having to write any {de,}serialization code.

Reading stuff saved on the current company is also very convenient, because you always can say `self.env.user.company_id`.

By now we've also a [convenient way](https://github.com/OCA/OCB/blob/12.0/odoo/addons/base/models/res_config.py#L371) to have a settings wizard write into config parameters, but those would be a little bit clumsy to use in https://github.com/OCA/partner-contact/pull/873/files#diff-adf307efb81bc5fecba0f8320c52202eR6